### PR TITLE
Fixing wandb related issues. Removing deprecation from private method

### PR DESCRIPTION
**Pull request by okuchaiev**
Signed-off-by: Oleksii Kuchaiev <okuchaiev@nvidia.com>

### CODE REVIEW

File: nemo/core/callbacks.py [hunk 1]
```diff
@@ -449,6 +449,8 @@ def on_action_start(self):
             if self._wandb_name is not None or self._wandb_project is not None:
                 if _WANDB_AVAILABLE and wandb.run is None:
                     wandb.init(name=self._wandb_name, project=self._wandb_project)
+                elif wandb.run is not None:
```
Comment (titu1994) [inline nemo/core/callbacks.py]: If _WANDB_AVAILABLE is false, then that means wandb is a None object, therefore wandb.run will throw an error.  Needs to check for the above flag prior to any operation on wandb

File: nemo/core/callbacks.py [hunk 2]
```diff
@@ -601,9 +603,10 @@ def on_action_start(self):
         if self.global_rank is None or self.global_rank == 0:
             if _WANDB_AVAILABLE and wandb.run is None:
                 wandb.init(name=self._name, project=self._project)
-
                 if self._args is not None:
                     wandb.config.update(self._args)
+            elif wandb.run is not None:
```
Comment (titu1994) [inline nemo/core/callbacks.py]: Same issue as above.

File: nemo/core/callbacks.py (hunk 1 above)
Comment (okuchaiev) [reply nemo/core/callbacks.py]: thanks! fixed

File: nemo/core/callbacks.py (hunk 2 above)
Comment (okuchaiev) [reply nemo/core/callbacks.py]: fixed

**Pull request merged**